Create ui-c8y-1022-25-0-update-monaco-and-ace-editors-highlight-colors-9846-graft-release-cd.md (#3670)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1022-25-0-update-monaco-and-ace-editors-highlight-colors-9846-graft-release-cd.md

---
date: ""

> title: update Monaco and Ace editors highlight colors

Reviewer: ```suggestion
title: Updated Monaco and Ace editor highlighting colors
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-63159
version: 1022.25.0
---

> Co-authored-by: Carlos Ceia <carlos.ceia@cumulocity.com>

Reviewer 2: ```suggestion
To improve the readability when working with the Monaco and Ace editors in the platform, the selection highlighting colors have been updated. This improvement applies to all areas where the Monaco and Ace editors are used, such as the application builder, smart rules, and device management.
```

> To improve the readability when working with the Monaco and Ace editors in the platform, the selection highlighting colors have been updated. This improvement applies to all areas where the Monaco and Ace editors are used, such as the application builder, smart rules, and device management.

Reviewer 3: ```suggestion
To improve the readability when working with the Monaco and Ace editors in the platform, the selection highlighting colors have been updated.This improvement applies to all areas where the Monaco and Ace editors are used, such as Administration (Branding and Localization), Cockpit (HTML widget, dashboard import/export), and Device Management.
```

> To improve the readability when working with the Monaco and Ace editors in the platform, the selection highlighting colors have been updated.This improvement applies to all areas where the Monaco and Ace editors are used, such as Administration (Branding and Localization), Cockpit (HTML widget, dashboard import/export), and Device Management.

Reviewer: ```suggestion
To improve the readability when working with the Monaco and Ace editors in the platform, the selection highlighting colors have been updated. This improvement applies to all areas where the Monaco and Ace editors are used, such as Administration (Branding and Localization), Cockpit (HTML widget, dashboard import/export), and Device Management.
```
